src/inject/generic/core/lyrics.js

```diff
@@ -28,6 +28,7 @@ window.addEventListener('load', () => {
   };
   // Handle time progression of a song
   const timeHandler = (timeObj) => {
+    $('#lyrics_back > #lyrics_progress > #lyrics_bar').width(`${timeObj.current / timeObj.total * 100}%`);
```

Comment:
It is entirely possible for `timeObj.total` to be `0`.  That needs to be handled somehow